addons/survey/tests/test_survey_invite.py

```diff
@@ -202,7 +202,7 @@ def test_survey_invite_email_from(self):
         action['context']['default_send_email'] = True
         invite_form = Form.from_action(self.env, action)
         invite_form.partner_ids.add(self.survey_user.partner_id)
-        invite_form.template_id.write({'email_from':'{{ object.partner_id.email_formatted }}'})
+        invite_form.template_id.sudo().write({'email_from': '{{ object.partner_id.email_formatted }}'})
```

Comment:
Why do we need sudo() here?

Reply:
Now we don't need that :), earlier it was necessary as the write to the template required accessing ir.model record.